cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiffTest.java

```diff
@@ -0,0 +1,281 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.operator.cluster.operator.resource;
+
+import io.strimzi.operator.cluster.KafkaVersionTestUtils;
+import io.strimzi.operator.cluster.model.KafkaVersion;
+import io.strimzi.test.TestUtils;
+import org.apache.kafka.clients.admin.AlterConfigOp;
+import org.apache.kafka.clients.admin.Config;
+import org.apache.kafka.clients.admin.ConfigEntry;
+import org.apache.kafka.common.config.ConfigResource;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+
+public class KafkaBrokerConfigurationDiffTest {
+
+    private static final KafkaVersion.Lookup VERSIONS = KafkaVersionTestUtils.getKafkaVersionLookup();
+    private static final String KAFKA_VERSION = "2.5.0";
+    KafkaVersion kafkaVersion = VERSIONS.version(KAFKA_VERSION);
+    private int brokerId = 0;
+
+    private String getDesiredConfiguration(List<ConfigEntry> additional) {
+        try (InputStream is = getClass().getClassLoader().getResourceAsStream("desired-kafka-broker.conf")) {
+            String desiredConfigString = TestUtils.readResource(is);
+
+            for (ConfigEntry ce : additional) {
+                desiredConfigString += "\n" + ce.name() + "=" + ce.value();
+            }
+
+            return desiredConfigString;
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Rethrow a `RuntimeException`. Honestly, `printStackTrace()` is never the right thing to do, especially in a test.